services/API-service/src/scripts/seed-admin-area-data.ts

```diff
@@ -85,38 +85,135 @@ export class SeedAdminAreaData implements InterfaceScript {
 
     // UGA
     if (envCountries.includes('UGA')) {
+      const countryCodeISO3 = 'UGA';
+      const adminlevel = 2;
       // Flood vulnerability
       const floodVulnerabilityFilename = `./src/scripts/git-lfs/admin-area-data/flood_vulnerability_UGA.csv`;
       const floodVulnerabilityData = await this.seedHelper.getCsvData(
         floodVulnerabilityFilename,
       );
       const floodVulnerabilityDataArray = floodVulnerabilityData.map(area => {
         return {
-          countryCodeISO3: 'UGA',
-          adminLevel: 2,
+          countryCodeISO3: countryCodeISO3,
+          adminLevel: adminlevel,
           placeCode: area['placeCode'],
           indicator: area['indicator'],
           value: area['value'],
         };
       });
-
       await this.adminAreaDataRepository.save(floodVulnerabilityDataArray);
 
       // COVID risk
       const covidRiskFilename = `./src/scripts/git-lfs/admin-area-data/covid_risk_UGA.csv`;
       const covidRiskData = await this.seedHelper.getCsvData(covidRiskFilename);
       const covidRiskDataArray = covidRiskData.map(area => {
         return {
-          countryCodeISO3: 'UGA',
-          adminLevel: 2,
+          countryCodeISO3: countryCodeISO3,
+          adminLevel: adminlevel,
           placeCode: area['placeCode'],
           indicator: area['indicator'],
           value: area['value'],
         };
       });
-
       await this.adminAreaDataRepository.save(covidRiskDataArray);
     }
+
+    // ETH
+    if (envCountries.includes('ETH')) {
+      const countryCodeISO3 = 'ETH';
+      const adminlevel = 3;
+      // malaria_risk
+      let fileName = `./src/scripts/git-lfs/admin-area-data/malaria_risk_ETH.csv`;
+      let data = await this.seedHelper.getCsvData(fileName);
+      let dataArray = data.map(area => {
```

Comment:
Not critical but it's advisable to use types when coding in typescript. [See example here](https://github.com/rodekruis/IBF-system/blob/master/services/API-service/src/scripts/seed-redcross-branches.ts#L8-L16). It helps ensure that the data is in the correct format and structure.